yourgov/components/FormMobileFoodVendorPermit/FormPageAlert.tsx

```diff
@@ -0,0 +1,53 @@
+import { type FieldError } from 'react-hook-form';
+import { useScrollToField } from '@ag.ds-next/react/field';
+import { ListItem, UnorderedList } from '@ag.ds-next/react/list';
+import { PageAlert } from '@ag.ds-next/react/page-alert';
+import { Text } from '@ag.ds-next/react/text';
+import { TextLink } from '@ag.ds-next/react/text-link';
+
+export function FormPageAlert({
+	errors,
+}: {
+	errors: {
+		[key: string]: {
+			message?: string;
+			from?: FieldError;
+			to?: FieldError;
+			value?: FieldError;
+			formatted?: FieldError;
+		};
+	};
+}) {
+	const scrollToField = useScrollToField();
+
+	// As our form schema contains nested objects, we are converting the errors from a nested object to a simple flat array
+	const flatErrors = Object.entries(errors)
+		.map(([key, value]) => {
+			if ('message' in value) return { key, message: value.message };
+			if ('from' in value) return { key, message: value.from?.message };
+			if ('to' in value) return { key, message: value.to?.message };
+		})
+		.filter((item): item is { key: string; message: string } =>
+			Boolean(item?.message)
+		);
+
+	return (
+		<PageAlert
+			focusOnMount
+			tabIndex={-1}
+			title="There is a problem"
```

Comment:
When this gets rendered in the steps, e.g. the "Business details" step, there's no `h2` so we're skipping from `h1` to `h3`. Maybe the `FormPageAlert` needs to have an `as` prop to pass to a custom `title` here? Something like:

```ts
// props
titleAs = "h2",
```

```suggestion
			title=<PageAlertTitle as={titleAs}>There is a problem</PageAlertTitle>
```

Reply:
Good pickup